### PR TITLE
KEA-145 As an admin, I want to upload/update a list of events so that users can see whole events

### PR DESCRIPTION
**Pull request by jiahn326**
KEA-145 As an admin, I want to upload/update a list of events so that users can see whole events

### CODE REVIEW

File: backend/tests/helpers/constructor.js [hunk 1]
```diff
@@ -107,26 +107,41 @@ const constructEvent = (id) => {
...
+                            "text": "Welcome!"
+                        }
+                    ]
                 }
             ],
-            "Description": "Event Description " + id,
             "HostingGroup": null,
-            "Fee": "$ Event Fee " + id, 
-            "Contact": "Event Contact " + id,
+            "Fee": null,
```
Comment (doolri1276) [inline backend/tests/helpers/constructor.js:143]: Why were these changed to null?

**Review by doolri1276 — CHANGES_REQUESTED** [review]

File: backend/config/sync/admin-role.admin-lrh30qnl.json
Comment (doolri1276) [inline backend/config/sync/admin-role.admin-lrh30qnl.json:1]: * Not related to this ticket

Looks like rebase is needed

**Review by doolri1276 — CHANGES_REQUESTED** [review]

File: backend/config/sync/admin-role.admin-lrh30qnl.json [hunk 2]
```diff
@@ -558,17 +573,6 @@
       },
       "conditions": []
     },
-    {
```
[the remark this answers is not in the document]
Comment (jiahn326) [reply backend/config/sync/admin-role.admin-lrh30qnl.json:561]: Reverted 👍

File: backend/config/sync/admin-role.strapi-super-admin.json
[the remark this answers is not in the document]
Comment (jiahn326) [reply backend/config/sync/admin-role.strapi-super-admin.json:1]: Reverted 👍

File: backend/config/sync/core-store.core_admin_project-settings.json
[the remark this answers is not in the document]
Comment (jiahn326) [reply backend/config/sync/core-store.core_admin_project-settings.json:1]: Reverted 👍

File: backend/src/components/common/address2.json
[the remark this answers is not in the document]
Comment (jiahn326) [reply backend/src/components/common/address2.json:1]: Thank you! I just updated address as well :)

File: backend/config/sync/admin-role.admin-lrh30qnl.json [hunk 3]
```diff
@@ -232,7 +232,12 @@
           "Private",
           "Location.Label",
           "Location.Address",
-          "Description"
+          "Location.City",
```
[the remark this answers is not in the document]
Comment (jiahn326) [reply backend/config/sync/admin-role.admin-lrh30qnl.json:235]: Can we have a call with this?

File: backend/config/sync/admin-role.admin-lrh30qnl.json [hunk 4]
```diff
@@ -232,7 +232,12 @@
...
+          "Location.State",
+          "Location.Country",
+          "Description",
+          "StartDate.TimeZone",
+          "EndDate.TimeZone"
         ],
         "locales": [
           "en"
```
[the remark this answers is not in the document]
Comment (jiahn326) [reply backend/config/sync/admin-role.admin-lrh30qnl.json:243]: Can we have a call with this?

File: backend/tests/api/event/index.js [hunk 5]
```diff
@@ -34,17 +34,22 @@ describe("Event Test", () => {
...
         await request(strapi.server.httpServer)
-            .post("/api/events")
+            .post("/api/events?populate=*")
             .set("accept", "application/json")
             .set("Authorization", `Bearer ${await jwt(adminUser.id)}`)
             .send(constructor.constructEvent(1))
             .expect("Content-Type", /json/)
             .expect(200)
             .expect((data) => {
+                console.log(data.body);
```
Comment (doolri1276) [inline backend/tests/api/event/index.js:44]: Could you remove the logs?

File: backend/config/sync/admin-role.admin-lrh30qnl.json
Comment (doolri1276) [reply backend/config/sync/admin-role.admin-lrh30qnl.json:1]: Rebase is still needed

File: backend/tests/helpers/constructor.js (hunk 1 above)
Comment (doolri1276) [reply backend/tests/helpers/constructor.js:143]: Can we have any test values to have it tested? Same thing goes for Contact.

File: backend/tests/api/event/index.js (hunk 5 above)
Comment (jiahn326) [reply backend/tests/api/event/index.js:44]: I removed. thank you!

File: backend/config/sync/admin-role.admin-lrh30qnl.json
Comment (jiahn326) [reply backend/config/sync/admin-role.admin-lrh30qnl.json:1]: I just did rebase. Could you please check if it is rebased well?

**Review by doolri1276 — APPROVED** [review]

**Pull request merged**